sde_collections/tasks.py

```diff
@@ -49,7 +52,7 @@ def _get_data_to_import(collection, server_name):
                 continue
 
             augmented_data = {
-                "model": "sde_collections.candidateurl",
+                "model": "sde_collections.url",
```

Comment:
bishwas tried to make this dump directly into dump url, but that didn't work. need to troubleshoot.